tests/eips/eip4844/test_excess_data_gas.py

```diff
@@ -204,11 +147,18 @@ def tx_max_fee(  # noqa: D103
 
 
 @pytest.fixture
-def tx_data_cost(  # noqa: D103
+def tx_max_fee_per_data(  # noqa: D103
```

Comment:
I noticed that this might have been duplicated. There is an equivalent fixture `tx_max_fee_per_data_gas` that I think can be deleted.

To keep things consistent with `blob_txs.py` I think we should rename the `tx_max_fee_per_data` fixture to `tx_max_fee_per_data_gas`. Delete the old `tx_max_fee_per_data_gas` fixture. And rename the `tx_max_fee` fixture to `tx_max_fee_per_gas`

Reply:
Great catch! I'll push a fix :)